client/boot/index.js

```diff
@@ -313,6 +310,10 @@ function boot() {
 
 	require( 'my-sites' )();
 
+	// clear notices
+	//TODO: remove this one when notices are reduxified - it is for old notices
+	page( '*', require( 'notices' ).clearNoticesOnNavigation );
+
 	if ( config.isEnabled( 'olark' ) ) {
```

Comment:
In order to make `displayOnNextPage` property to work as expected. If we don't do this we clear the notices we render the scheduled ones before rendering "mysites"